README.md

```diff
@@ -1351,6 +1345,10 @@ drive report-issue -title "Can't open my file" -body "Drive trips out every time
 cat bugReport.txt | drive issue -piped -title "push: dump on pushing from this directory"
 ```
 
+### Automating
```

Comment:
Do you agree on this version? I have dropped the "Automating" paragraph and reworded a little your proposal.